src/applications/ask-va/components/ReviewCollapsibleChapter.jsx

```diff
@@ -230,14 +230,17 @@ class ReviewCollapsibleChapter extends React.Component {
                   );
                 }}
                 buttonText="Save"
-                buttonClass="usa-button-primary"
+                buttonClass="usa-button-primary vads-u-width--auto"
                 ariaLabel={ariaLabel}
               />
               <button
                 aria-label="Cancel"
                 type="button"
                 id="cancel"
-                className=""
+                className="usa-button-secondary vads-u-width--auto"
```

Comment:
Just a head's up that you might be able to use the `va-button` secondary variation here instead of having to manage your own markup: 

https://design.va.gov/storybook/?path=/docs/uswds-va-button--docs#secondary